examples/multifile-mutation/dune

```diff
@@ -0,0 +1,2 @@
+(coqlib
+ (name multifile_mutation))
```

Comment:
What does this file do? Do I need to add it to all examples?

Reply:
This declares the test lib to compile, however it is still not useful as we cannot depend on the quickChick lib itself.

So it is unused at the moment.